signal_scanner_bot/signal.py

```diff
@@ -45,6 +44,45 @@ def message_timestamp(data: Dict) -> datetime:
     return dt
 
 
+def list_identities() -> str:
+    """
+        Function that calls the signal-cli `listIdentities` command and returns the entire result as a string
+    """
+    with env.SIGNAL_LOCK:
+        proc = subprocess.run(
+            [
+                "signal-cli",
+                "-u",
+                str(env.BOT_NUMBER),
+                "listIdentities"
+            ],
+            capture_output=True,
```

Comment:
I'll have to check that we're doing this elsewhere...but if you add `encoding='utf-8'` to this call, you shouldn't need to decode the stdout/stderr below!